Prepare release 1.0.0-rc.0 (#65)
GPP-Woo/GPP-zoeken: docs/configuration/services.rst

.. _configuration_services:

Services
========

> GPP-Publicatiebank is designed to operate in a micro-service environment, which broadly
> means that it talks to other APIs to be able to function correctly - see
> :ref:`installation_requirements` for some more background information and things that
> need to be arranged in those external services.

Reviewer: ```suggestion
GPP-zoeken is designed to operate in a micro-service environment, which broadly
means that it talks to other APIs to be able to function correctly - see
:ref:`installation_requirements` for some more background information and things that
need to be arranged in those external services.
```

> The services configuration in GPP-Publicatiebank stores the connection and
> authentication details for those external services.

Reviewer: ```suggestion
The services configuration in GPP-zoeken stores the connection and
authentication details for those external services.
```

General instructions
--------------------

To view and manage services, navigate to **Admin** > **Configuration** > **Services**.

The list shows an overview of configured services, displaying the label/name, the kind
of service and the base URL where it is hosted.

In the top right of the page, you can click the **Service toevoegen** button to add a
new service. To edit an existing service, click its label, which takes you to the
edit page.

> GPP-zoeken configuration

Reviewer: ```suggestion
GPP-publicatiebank configuration
```

> GPP-Publicatiebank configuration

Reviewer: ```suggestion
GPP-zoeken configuration
```

--------------------------------

> GPP-Zoeken depends on the `GPP-Publicatiebank <https://gpp-publicatiebank.readthedocs.io/>`_
> service to retrieve download urls to index the contents of documents. A service needs to be

Reviewer: ```suggestion
GPP-Zoeken depends on the `GPP-Publicatiebank <https://gpp-publicatiebank.readthedocs.io/>`_
service to retrieve the contents of referenced documents. A service needs to be
```
